packages/react-utilities/src/compose/README.md

```diff
@@ -211,69 +161,77 @@ const renderButton = state => {
 };
 ```
 
-## API reference
+#### Memoization of context values
```

Comment:
we also have cases where `react-context-selector` is used with a hook, but not memoized. Is this pattern we follow only for memoized context values or all context values ?

Reply:
I can add a section about `react-context-selector` if there is a sense. WDYT?

Reply:
Yeah a note about react-context-selector would be nice, there is an argument to do this for all contexts, but for primitive contexts it might be overkill... not sure how strict we want to be about consistency here

Reply:
Added in 1786ccd